src/browser/api/system.js

```diff
@@ -19,6 +19,7 @@ const fs = require('fs');
 const os = require('os');
 const electron = require('electron');
 const electronApp = electron.app;
+const electronBrowserWindow = electron.BrowserWindow;
```

Comment:
Although "consistency", I vote for "future" code, which is `import` here, just my opinion

Reply:
Also, IMO, should be a shorter name `import { BrowserWindow } from electron;`

Reply:
We can do `import { app, BrowserWindow } from electron;` instead of 3 lines of code

Reply:
existing code below, we should discuss if it's better to have a new JIRA for such code change/improvement.... Also, not a bad idea to have electronApp instead of app, so we don't get confused if it's OpenFin app or Electron app......

```
const electron = require('electron');
const electronApp = electron.app;
const ResourceFetcher = electron.resourceFetcher;
const session = electron.session;
const shell = electron.shell;
const { crashReporter } = electron;

```